pages/checkOut.tsx

```diff
@@ -1,5 +1,41 @@
-export default function CheckOutPage() {
+import CheckInOutForm from 'components/CheckInOutForm'
+import {
+  Box,
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  Typography,
+  Unstable_Grid2 as Grid2,
+  useMediaQuery,
+  useTheme,
+} from '@mui/material'
+
+export default function CheckInPage() {
+  const theme = useTheme()
+  const isMobileView = useMediaQuery(theme.breakpoints.down('sm'))
   return (
-    <h1>Check Out</h1>
+    <Grid2 container my={2} sx={{ flexGrow: 1 }}>
+      <Grid2 xs={12} sm={8} lg={6} smOffset={2} lgOffset={3}>
+        <Card variant={isMobileView ? 'elevation' : 'outlined'} elevation={0}>
+          <Box display="flex" flexDirection="column">
+            <CardContent sx={{ p: isMobileView ? 0 : 2 }}>
+              <Typography variant="h5" sx={{ mb: 2 }}>
+                Check in items
+              </Typography>
+              <CheckInOutForm
+                kioskMode={true}
+                users={[]}
+                itemDefinitions={[]}
+              />
+            </CardContent>
+
+            <CardActions sx={{ alignSelf: { xs: 'end' } }}>
```

Comment:
Add `mt: { xs: 1, sm: 0 }` so that there's a bit more space between the quantity component and the checkout button on mobile

![image](https://user-images.githubusercontent.com/71574118/221329598-bc72610c-a679-4344-875b-598dc0630989.png)

Reply:
I agree, it looks better with the margin. Resolved with 3b05ab6